src/azure-cli/azure/cli/command_modules/appservice/_params.py

```diff
@@ -356,6 +358,9 @@ def load_arguments(self, _):
                 c.ignore('reserved_instance_count')
                 c.argument('runtime', help="Canonicalized web runtime in the format of Framework:Version, e.g. \"PHP:7.2\"."
                                            "Use `az webapp list-runtimes` for available list")
+                c.argument('acr_use_identity', arg_type=get_three_state_flag(return_label=True), help="Enable or disable pull image from acr use managed identity")
```

Comment:
```suggestion
                c.argument('enable_acr_identity', arg_type=get_three_state_flag(return_label=True), help="Enable or disable pull image from acr use managed identity")
```

Reply:
Hi @zhoxing-ms, we already have --acr-use-identity in az webapp create, so we are using this to keep parameters in different commands consistent

Reply:
Got it, thanks~